.github/workflows/main.yml

```
@@ -4,38 +4,30 @@ on:
   push:
   pull_request:
   workflow_dispatch:
+  release:
+    types: ['published', 'released']
   schedule:
     # build at least once a month
     - cron: '0 0 1 * *'
 
 jobs:
-  appimage-x86_64:
-    name: AppImage x86_64
+  appimage:
+    strategy:
+      matrix:
+        include:
+          - arch: 'x86_64'
+          - arch: 'i386'
```

I prefer upper-case names that match the environment variable name.

Also, you can do a little shorter:

```suggestion
          - ARCH: ["x86_64", "i386"]
```